packages/interpolate-components/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.1]
+
+### Changed
+- @types/react peer dependency increased from >=16.2.0 to >=16.14.23
```

Comment:
This happened automatically during renovate updates. We probably don't need to publish it, but since i18n-calypso depends on this package, I think it makes sense.

Reply:
This can be disruptive… if there's no reason to change this peerDep (it sounds like there isn't) it's probably better to revert it instead of changing it.

https://github.com/Automattic/wp-calypso/pull/57564 widened this back to 16.2 specifically.